content/management/changelog/index.mdx

```diff
@@ -19,6 +19,9 @@ Since the migration is still ongoing, different parts will be successively moved
 
 - The [**Delete Bot**](/management/configuration-api/v3.3/#delete-bot) method had the `bot_agent_id` parameter renamed to `id`.
 - The [**Get Bot**](/management/configuration-api/v3.3/#get-bot) method had the `bot_agent_id` parameter renamed to `id`.
+- The [**Create Bot**](/management/configuration-api/v3.3/#create-bot) method has new parameter, `timezone`. It's required when used with `work_scheduler`; otherwise optional.
```

Comment:
```suggestion
- The [**Create Bot**](/management/configuration-api/v3.3/#create-bot) method has a new parameter, `timezone`. It's required when used with `work_scheduler`; otherwise optional.
```